jopa-impl/src/main/java/cz/cvut/kbss/jopa/oom/DataPropertyFieldStrategy.java

```diff
@@ -43,6 +44,14 @@ boolean isValidRange(Object value) {
     }
 
     boolean canBeConverted(Object value) {
+        if(attribute.getJavaType() != null && attribute.getJavaType().isPrimitive()) {
```

Comment:
Sadly, this check cannot be removed so simply:
```bash
[ERROR]   PluralAnnotationPropertyStrategyTest.addValueFromAxiomAcceptsIdentifiersForLexicalFormAttribute:244 » NullPointer Cannot invoke "java.lang.Class.isPrimitive()" because the return value of "cz.cvut.kbss.jopa.model.metamodel.AbstractAttribute.getJavaType()" is null
[ERROR]   PluralAnnotationPropertyStrategyTest.addValueFromAxiomAddsStringToUriValues:131 » NullPointer Cannot invoke "java.lang.Class.isPrimitive()" because the return value of "cz.cvut.kbss.jopa.model.metamodel.AbstractAttribute.getJavaType()" is null
[ERROR]   PluralAnnotationPropertyStrategyTest.addValueFromAxiomTransformsValueToLexicalForm:221 » NullPointer Cannot invoke "java.lang.Class.isPrimitive()" because the return value of "cz.cvut.kbss.jopa.model.metamodel.AbstractAttribute.getJavaType()" is null
```